partcad/src/partcad/part_factory_scad.py

```
@@ -12,15 +12,24 @@
+from .runtime_python import PythonRuntime
+
+from . import wrapper
+from ocp_serialize import register as register_ocp_helper
 from . import logging as pc_logging
 
 
 class PartFactoryScad(PartFactoryFile):
+    runtime: PythonRuntime
+    cwd: str
+
     def __init__(self, ctx, source_project, target_project, config, can_create=False):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)